src/ReactVapor.ts

```diff
@@ -70,6 +71,7 @@ export interface IReactVaporState {
     toastContainers?: IToastsState[];
     tableHeaderCells?: ITableHeaderCellsState;
     tables?: ITablesState;
+    tableHOCHeader?: ITableWithSortState[];

```

Comment:
As our state is getting bigger and bigger, what do you think of grouping related states under the same key. For example we could have a `tableHOC` key that contains all `tableHOC*` states?

Reply:
I think it's better to keep things separated, the problem if we use the same key is that all reducers for this key will run which add some overhead